doc/api/crypto.md

```diff
@@ -1170,15 +1170,16 @@ added: v6.0.0
 Property for checking and controlling whether a FIPS compliant crypto provider is
 currently in use. Setting to true requires a FIPS build of Node.js.
 
-### crypto.createCipher(algorithm, password)
+### crypto.createCipher(algorithm, password[, options])
 <!-- YAML
 added: v0.1.94
 -->
 - `algorithm` {string}
 - `password` {string | Buffer | TypedArray | DataView}
+- `options` {object} combined [OpenSSL options][] and [`stream.transform` options][]
```

Comment:
The `options` object isn't actually combined, but only used for streams:

https://github.com/nodejs/node/blob/0d22858d67f5f8f7959a55ceca23adafe12827d5/lib/crypto.js#L136-L145

The first instance of `options` is used when the function is not called with `new` (it's passed back into itself).

Same goes for the other classes.